code/modules/admin/admin.dm

```diff
@@ -3313,6 +3313,7 @@ var/global/noir = 0
 				switch(href_list["type"])
 					if("check_antagonist")
 						if (ticker?.mode && current_state >= GAME_STATE_PLAYING)
+							#define isdeadantag(M) (isdead(M) || (isVRghost(M) || isghostcritter(M) || inafterlife(M) || isghostdrone(M)))
```

Comment:
...antag?

Reply:
I guess it does also include heads etc, maybe "role" or "player" is better.